createdisk.sh

```diff
@@ -347,6 +347,9 @@ internalIP=$(${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'hostname -i')
 # Add a dummy network interface with internalIP
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- "sudo nmcli conn add type dummy ifname eth10 con-name dummy ip4 ${internalIP}/24  && sudo nmcli conn up dummy"
 
+# Add internalIP as node IP for kubelet systemd unit file
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- "sudo sed -i.back '/kubelet /a\      --node-ip="$ip" \\\' /etc/systemd/system/kubelet.service"
+
```

Comment:
> This will make sure when kubelet service run then node internal IP would be what defined here instead of detecting it automatic.
"service runs ... what is defined here ... automatically"